src/tests/countingCards.test.js

```diff
@@ -0,0 +1,28 @@
+// let count = require("../challenges/countingCards.js")
+const cc = require("../challenges/countingCards.js")
+const cardGame = require("../challenges/countingCards.js")
+
+// const cases = [
+//   [2, "1 Bet"],
+//   [3, "2 Bet"],
+//   [4, "3 Bet"],
+//   [5, "4 Bet"],
+//   [6, "5 Bet"],
+// ]
+
+// describe(" cc2 cc3 cc4 cc5 cc6", () => {
+//   test.each(cases)(
+//     "given %p as the card, returns %p",
+//     (firstArg, expectedResult) => {
+//       const result = cc(firstArg)
+//       expectedResult(result).toEqual(expectedResult)
+//     }
+//   )
+// })
+test("cc2 cc3 cc4 cc5 cc6", () => {
+  expect((cc(2), cc(3), cc(4))).toBe("3 Bet")
+})
+
+test("cc5", () => {
```

Comment:
cc(5) doesn't pass the test the way i thought it should and im not sure why.

Reply:
Couple thoughts.  1 you shouldn't write your tests with the expectations that they run in a certain order. Most frameworks allow running a single test and this should always work for a test.  Also also of frameworks run the tests in parallel to speed them up, and some run the ones that failed first.  So in general every test should have everything it needs stand alone.

It would better practice to avoid the use of a Global variable to meet this end, but it's kinda a part of the code challenge.  If you need to use the global use a different one for each of you functions, or move the functions to seperate files.